lib/configuration_options.dart

```diff
@@ -89,6 +89,13 @@ const String iOSAllowToolbarTitleChange = "allowToolbarTitleChange";
 const String toolbarTitle = "toolbarTitle";
 
 const String androidSettingsMenuItems = "settingsMenuItems";
+const String iOSSettingsMenuItems = "settingsMenuItems";
 
 const String showActionNavigationButtons = "showActionNavigationButtons";
 const String iOSShowActionNavigationButtonLabels = "iOSShowActionNavigationButtonLabels";
+
+const String pageLayoutMode = "pageLayoutMode";
+const String pageLayoutModeAutomatic = "automatic";
+const String pageLayoutModeSingle = "single";
+const String pageLayoutModeDouble = "double";
+const String iOSIsFirstPageAlwaysSingle = "isFirstPageAlwaysSingle";
```

Comment:
Are there reasons to make this configuration option iOS specific?
Otherwise, I would extend it Android as well https://pspdfkit.com/api/android/reference/com/pspdfkit/configuration/PdfConfiguration.html#isFirstPageAlwaysSingle()

It would be an easy fix, I can do that: `iOSIsFirstPageAlwaysSingle` would become `isFirstPageAlwaysSingle`.

Reply:
I wasn't sure if Android had this property, so I prefixed it with `iOS`. If we have this property on Android, then I'm all for renaming it as long as we implement it on Android too :)

Reply:
I've also just updated the Acceptance Criteria.